tests/ocean/test_ocean_assets.py

```diff
@@ -93,7 +93,6 @@ def _get_num_assets(_minter):
     assert ddo.metadata["main"]["name"] == _name
     with pytest.raises(ValueError):
         ocn.assets.update(ddo, bob)
-        # assert False, "this asset update should fail, but did not."
```

Comment:
Why is this removed?

Perhaps we should go in the other direction: uncomment the line and to revive the effectiveness of the test?

Reply:
pytest.raises will take care of that, I've tested it both ways. If a ValueError is not thrown in this block, the test fails.

Reply:
and it's consistent with previous blocks. This was the only one done using try/except.